akka-actor/src/main/resources/reference.conf

```diff
@@ -25,6 +25,17 @@ akka {
   # (akka.actor.ActorSystem.Settings, akka.event.EventStream) parameters.
   logging-filter = "akka.event.DefaultLoggingFilter"
 
+  # Specifies the default loggers dispatcher
+  loggers-dispatcher = "akka.loggers.pinned-dispatcher"
```

Comment:
Hmmm, too many threads? I understand what you are saying but ... it is an I/O thread, I wanted to make it use akka.io.dispatcher but then I would at being messing with the `selector` functionality whatever that is ;-)